server/game/gameSystems/DiscardCardsFromHandSystem.ts

```diff
@@ -17,7 +17,7 @@ export interface IDiscardCardsFromHandProperties extends IPlayerTargetSystemProp
     this may be necessary for a player discarding from their own ahnds if a card condition or filter exits to keep them honest */
     cardTypeFilter?: CardTypeFilter | CardTypeFilter[];
     cardCondition?: (card: Card, context: AbilityContext) => boolean;
-    discardingRelativePlayer?: RelativePlayer;
+    choosingRelativePlayer: RelativePlayer;
```

Comment:
Message in discord as well but leaving it here. Small favor to ask - seeing this here jogged my memory that we have a similar parameter for target resolvers but it's named choosingPlayer: https://github.com/SWU-Karabast/forceteki/blob/8e330598f15e55c64c1db6526a7f8b97ad9a8e09/server/game/TargetInterfaces.ts#L73

I think it's similar enough in usage to this one that we should make the names the same, I'm fine with either choosingRelativePlayer or choosingPlayer for everything

Reply:
Decided to change the new one to choosingPlayer, as the other version is in many many spots.  Didn't seem worth changing that one